classes/UpgradeTools/CoreUpgrader/CoreUpgrader17.php

```diff
@@ -87,7 +90,33 @@ protected function upgradeLanguage($lang)
         \Language::installSfLanguagePack($lang_pack['locale'], $errorsLanguage);
 
         if (!$this->container->getUpgradeConfiguration()->shouldKeepMails()) {
-            \Language::installEmailsLanguagePack($lang_pack, $errorsLanguage);
+            $mailTheme = \Configuration::get('PS_MAIL_THEME', null, null, null, 'modern');
```

Comment:
Should we handle backward compatibility and use the `Language::installEmailsLanguagePack` for versions older than the 1.7.6? If so maybe this should be extracted in a dedicated tool class or service.

Reply:
I'm in favor to keep the backward compatibilty!

Reply:
Yes keep it